oca_dependencies.txt

```diff
@@ -1 +1,2 @@
-server-tools
\ No newline at end of file
+web https://github.com/hbrunn/web 10.0-web_drop_target
```

Comment:
Please remove this and add `web` now that the dependency has been merged

Reply:
done